docs/api/spaces-management/resolve_copy_saved_objects_conflicts.asciidoc

```diff
@@ -25,51 +25,86 @@ Execute the <<spaces-api-copy-saved-objects,copy saved objects to space API>>, w
 `space_id`::
 (Optional, string) The ID of the space that contains the saved objects you want to copy. When `space_id` is unspecified in the URL, the default space is used. The `space_id` must be the same value used during the failed <<spaces-api-copy-saved-objects, copy saved objects to space API>> operation.
 
+[role="child_attributes"]
 [[spaces-api-resolve-copy-saved-objects-conflicts-request-body]]
 ==== {api-request-body-title}
 
 `objects`::
   (Required, object array) The saved objects to copy. The `objects` must be the same values used during the failed <<spaces-api-copy-saved-objects, copy saved objects to space API>> operation.
+
+.Properties of `objects`
+[%collapsible%open]
+=====
   `type`:::
     (Required, string) The saved object type.
+
   `id`:::
     (Required, string) The saved object ID.
+=====
 
 `includeReferences`::
   (Optional, boolean) When set to `true`, all saved objects related to the specified saved objects are copied into the target spaces. The `includeReferences` must be the same values used during the failed <<spaces-api-copy-saved-objects, copy saved objects to space API>> operation. The default value is `false`.
 
 `retries`::
   (Required, object) The retry operations to attempt. Object keys represent the target space IDs.
+
+.Properties of `retries`
+[%collapsible%open]
+=====
   `<space_id>`:::
   (Required, array) The errors to resolve for the specified `<space_id>`.
+=====
```

Comment:
If the retries object contains <space_id> which contains type (etc), I think this section is ending too soon:
```suggestion
+
```